src/render/index.js

```diff
@@ -16,7 +20,9 @@ class App extends React.PureComponent {
       componentPropMap: new Map(),
       ast: undefined,
       datasets: undefined,
-      currentSidebarNode: null
+      currentSidebarNode: null,
+      url: '',
+      currProcess: null
```

Comment:
Nitpick: let's try to stick with full word variable names (or at least keep them consistent when they're abbreviated `curr` vs `current`).

Reply:
Gotcha I'll make sure this is fixed. Should I push up these changes on this branch and make another PR or fix them in the next issue?

Reply:
Including in the next issue is fine, I don't think it needs its own review